source/develop/release-management/features/virt/virt-v2v-integration.html.md

```diff
@@ -139,6 +139,33 @@ Import from external environments using virt-v2v in terms of the general flow:
 
 10. Unlock VM & disks
 
+##### Import OVA from VMware
+
+Currently oVirt support importing OVA only from VMware hypervisor.
+The OVA formats that oVirt supported are tar, zip and folder.
```

Comment:
ok, so we can leave the zip option then but let's add a statement that it is not part of the ova specification and thus deprecated (as we do want to be as close as possible to the specification in 4.1)